src/lib/PWM/PWM_ESP32.cpp

```diff
@@ -128,7 +137,10 @@ pwm_channel_t PWMController::allocate(uint8_t pin, uint32_t frequency)
             .duty_mode = MCPWM_DUTY_MODE_0,
             .counter_mode = MCPWM_UP_COUNTER,
         };
-        mcpwm_gpio_init(mcpwm_config[channel].unit, mcpwm_config[channel].signal, pin);
+        if (mcpwm_gpio_init(mcpwm_config[channel].unit, mcpwm_config[channel].signal, pin) != ESP_OK)
+        {
+            DBGLN("failed %d", pin);
```

Comment:
Maybe add a tiny bit more info here? Seeing just `failed 7` in a log might be hard to track down. Throw a `mcpwm_gpio_init` in front?